tests/Lens.spec.ts

```diff
@@ -67,17 +66,11 @@ const setup = async (
     args: [[ONE_ETHER], [ONE_ETHER], [ONE_ETHER], [ONE_ETHER]],
   })
 
-  const incentivesController = await deployIncentivesController({
-    api,
-    signer: deployer,
-    args: [],
-  })
-
```

Comment:
Why should this and the subsequent following be deleted?

```diff
    args: [
-      incentivesController.address,
+      null,
```

Reply:
incentiveController is optional and it doesn't affect.

Reply:
I just tried to test if removing incentives controller can remove timeout error in e2e tests.